new-dti-website/src/app/apply/page.tsx

```diff
@@ -17,7 +17,7 @@ const ApplyHero = () => {
     <section id="Apply Hero" className="text-[#FEFEFE] flex items-center relative">
       {!isApplicationOpen && (
         <Banner
-          message={`We're no longer accepting applications for ${config.semester}. Stay tuned for opportunities next semester!`}
+          message={`Applications for ${config.semester} are not open yet. Stay tuned for opportunities around the beginning of the semester!`}
```

Comment:
`config.semester` is still Spring 2025. Let's just change it here instead of config, so it doesn't mess up anything where it's used.
```suggestion
          message={`Applications for Fall 2025 are not open yet. Stay tuned for opportunities around the beginning of the semester!`}
```

Reply:
okie